frontend/messages/en/PrivacyPolicy.json

```diff
@@ -91,7 +85,7 @@
       },
```

Comment:
Please remove the "request" from this as well. It is also an option in the settings page and doesn't require an explicit request.